ocaml/rrdd/rrdd_main.ml

```diff
@@ -481,7 +481,12 @@ let update_vbds doms =
 					 * when CAR-133 is in place. CAR-133 attempts to plug raw VDI's
 					 * directly from LVs to blkback. *)
 					let inflight = Int64.add (Int64.sub b.st_rd_req b.st_rd_cnt) (Int64.sub b.st_wr_req b.st_wr_cnt) in
-					let iowait = Int64.to_float (Int64.div (Int64.add b.st_rd_cnt b.st_wr_cnt) (Int64.add b.st_rd_sum_usecs b.st_wr_sum_usecs))  /. 1000. in
+					let iowait =
+						let cum_wr_rd_usecs = Int64.add b.st_rd_sum_usecs b.st_wr_sum_usecs in
```

Comment:
After talking about this: don't bother with infix operators as this is in an old code base that we are not carrying forward.